menas/src/main/scala/za/co/absa/enceladus/menas/controllers/VersionedModelController.scala

```diff
@@ -158,4 +158,17 @@ abstract class VersionedModelController[C <: VersionedModel with Product with Au
     versionedModelService.disableVersion(name, v)
   }
 
+  @PutMapping(Array("/lock/{name}"))
+  @ResponseStatus(HttpStatus.OK)
+  def lock(@PathVariable name: String): CompletableFuture[UpdateResult] = {
```

Comment:
Please make it limited to admin roles only. As the property definitions are in `PropertyDefinitionController`
```scala
  @PreAuthorize("@authConstants.hasAdminRole(authentication)")
```

Reply:
My reason for asking this was also because disable now is not limited to admin. Should it be the case as well?

Reply:
Admin came much later and was not considered/forgotten for delete. And see the bonus note bellow 😉